test/actions.spec.js

```diff
@@ -64,4 +64,52 @@ describe('actions.js', () => {
       expect(JavaScriptExecution.createTimer).toHaveBeenCalledWith(identifier, instant);
     });
   });
+
+  describe('Transformation', () => {
+    describe('transform', () => {
+      const type = 'MAP';
+      const fn = 'en.map';
+      const value = 'ON';
+
+      it('throws TypeError when a wrong argument is passed.', () => {
+        expect(() => Transformation.transform({}, fn, value)).toThrow(TypeError);
+        expect(() => Transformation.transform(type, {}, value)).toThrow(TypeError);
+        expect(() => Transformation.transform(type, fn, { value })).toThrow(TypeError);
+        expect(JavaTransformation.transform).not.toHaveBeenCalled();
+      });
+
+      it('delegates to Java Transformation.', () => {
+        const type = 'MAP';
```

Comment:
You can omit the definition of `type`, `fn` and `value` as they are already set in the enclosing `describe` block. Same on lines 105 – 107.